catroid/src/main/java/org/catrobat/catroid/common/Constants.java

```diff
@@ -33,7 +33,7 @@
 	// Reflection in testcases needed
 	// http://stackoverflow.com/questions/1615163/modifying-final-fields-in-java?answertab=votes#tab-top
 
-	public static final float CURRENT_CATROBAT_LANGUAGE_VERSION = Float.valueOf(0.993f);
+	public static final float CURRENT_CATROBAT_LANGUAGE_VERSION = Float.valueOf(0.994f);
```

Comment:
doesn't this need a conversion action from older language version programs in `ProjectManager.loadProject()`?